docs/tutorials/subscription-platform.md

```diff
@@ -377,6 +378,72 @@ After the initial payment during subscription creation, the recurring payments f
+
+### Environments
+
+The Subscription Platform makes use of 3 Contentful [environment aliases](https://www.contentful.com/developers/docs/concepts/environment-aliases/). The environment aliases are setup to match the current supported Stripe environments, and are listed below.
+
+* Prod (master)
+  * Production environment
+  * Stripe environment: Subscription Platform
+  * Refers to the Contentful “master” environment alias.
+* Stage
+  * Stage and QA environment
+  * Stripe environment: SUB_PLAT_STAGE
+* Dev
+  * Development
+  * Stripe environment: SUB_PLAT_DEV
+
+### Deployments
+
+Deployments will be handled manually and will make use of the Merge app by Contentful, following the steps discussed in the tutorial, [Merge content type changes with Merge app](https://www.contentful.com/developers/docs/tutorials/general/merge-app/).
+
+* For each merge, ensure to complete the optional step "Export migration script" and store the migration scripts in [the FxA monorepo](https://github.com/mozilla/fxa/). (##TODO: Add URL to migration script directory)
+
+:::caution
+Changes made in the Appearance tab of a field of a content type, are not merged when using the Merge app, and therefore also not included in the migration script.
+:::
+
+#### Typical changes
+
+Changes are made in the Dev environment and follow the deployment path as shown below.
+
+```mermaid
+flowchart LR
+    Dev --> Stage --> Prod
+```
+
+For a typical change and deployment, the following steps should be taken.
+
+1. Make changes in the `Dev` environment
+1. After a regular FxA release has been deployed to Stage, using the Merge App, merge changes from the `Dev` environtment alias to the `Stage` environment alias.
+1. Similarly, after a regular FxA release has been deployed to Production, using the Merge App, merge changes from the `Stage` environtment alias to the `Prod` environment alias.
+
+#### Sensitive or large content model changes
+
+In cases where many changes are made or a technically sensitive change that could introduce breaking changes, it is recommended to make the changes in a new environment, for example called `Feature`, before merging those changes to `Dev`.
+
+The resulting deployment path would be as follows.
+
+```mermaid
+flowchart LR
+    Feature --> Dev --> Stage --> Prod
+```
+
+For a sensitive or large content model change and deployment, the following steps should be taken.
+
+1. (Optional) Delete the existing Feature environment. (Check with Subscription Platform team first before doing this)
+1. Create new `Feature` environment from `Dev` alias environment
+1. Make changes in `Feature` environment
+1. Merge changes to `Dev` and follow typical deploymant path
+
+Once the changes in the `Feature` environment have been merged into `Dev`, the `Feature` environment can be deleted.
+
+#### Hotfixes
+
+Hotfixes should follow either of the release paths discussed above, however mergers to the `Stage` and `Prod` do not need to follow the FxA release schedule.
```

Comment:
- change "mergers" to "merges."
- "the" can be removed before "...`Stage` and `Prod`..."
- Similar to above (see other comments), remove code blocks on "Stage" and "Prod" and change "Prod" to "Production" for consistency. Alternatively, wrap all in code blocks and change all instances of "Production" to "Prod."